H/HYPRE/build_tarballs.jl

```diff
@@ -47,25 +53,46 @@ cmake .. \
 
 make -j${nproc}
 make install
+"""
 
+augment_platform_block = """
+    using Base.BinaryPlatforms
+    $(MPI.augment)
+    augment_platform!(platform::Platform) = augment_mpi!(platform)
 """
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
 platforms = supported_platforms(; experimental=true)
 
+# MPItrampoline requires Fortran
+platforms = expand_gfortran_versions(platforms)
```

Comment:
Same comment as https://github.com/JuliaPackaging/Yggdrasil/pull/5140#discussion_r919457600

Reply:
I am trying this.

Reply:
Leaving out this statement leads to this error:
```
[ Info: /agent/_work/1/s/H/HYPRE/build/x86_64-linux-gnu-mpi+mpitrampoline/MZd0jNTD/x86_64-linux-gnu-libgfortran3-cxx03-mpi+mpitrampoline/destdir/lib/libHYPRE.so cannot be dlopen'ed
```

See e.g. https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=20647&view=logs&j=7aeb1989-ab2c-51b9-49d6-09383b673c9a&t=f7086e79-1917-5649-bdc5-1508d4a22ae4

Reply:
Please advise how to proceed. Should we debug this, or just put the expansion back in?

Reply:
Isn't this _precisely_ the same situation as https://github.com/JuliaPackaging/Yggdrasil/pull/5028#issuecomment-1165869962?

Reply:
Yes!